tests/test_flatten.py

```diff
@@ -31,3 +45,60 @@ def test_flatten_dict():
             "contextInfo_context2": "info2",
         }
     ]
+
+
+def test_flatten_pd():
+    flattened_pd = flatten(
+        PD_DATA, nested_columns=["contextInfo", "neighborhood"]
+    )
+    assert flattened_pd.equals(
+        DataFrame(
+            [
+                {
+                    "answer": 42,
+                    "contextInfo_context1": "info1",
+                    "contextInfo_context2": "info2",
+                }
+            ]
+        )
+    )
+
+
+def test_flatten_df_is_called():
+    with patch("crossfire._flatten_df", autospec=True) as mock_flatten_df:
+        mock_flatten_df.return_value = Series(
+            {
+                "contextInfo_contextInfo_context1": "info1",
+                "contextInfo_contextInfo_context2": "info2",
```

Comment:
Why does `contextInfo` appear twice here?

Wasn't it supposed to be:

```suggestion
                "contextInfo_context1": "info1",
                "contextInfo_context2": "info2",
```